pkg/controller/servicing/lifecycler.go

```diff
@@ -173,15 +178,53 @@ func (lc *NodeLifeCycler) Replace(node *core_v1.Node) error {
 
 // Uncordon removes the updating annotation and uncordons the node
 func (lc *NodeLifeCycler) Uncordon(node *core_v1.Node) error {
-	delete(node.Annotations, AnnotationUpdateTimestamp)
-	_, err := lc.Kubernetes.CoreV1().Nodes().Update(node)
-	return err
+	if err := lc.removeUpdatingAnnotation(node); err != nil {
+		return errors.Wrap(err, "failed to uncordon node")
+	}
+	if err := drain.Uncordon(lc.Kubernetes.Core().Nodes(), node, lc.Logger); err != nil {
+		return errors.Wrap(err, "failed to uncordon node")
+	}
+	return nil
 }
 
 func (lc *NodeLifeCycler) setUpdatingAnnotation(node *core_v1.Node) error {
-	node.Annotations[AnnotationUpdateTimestamp] = Now().UTC().Format(time.RFC3339)
-	_, err := lc.Kubernetes.CoreV1().Nodes().Update(node)
-	return err
+	copy := node.DeepCopy()
+	copy.Annotations[AnnotationUpdateTimestamp] = Now().UTC().Format(time.RFC3339)
+	err := lc.patch(node, copy)
+	if err != nil {
+		errors.Wrap(err, "failed to set updating annotation")
+	}
+	return nil
+}
+
+func (lc *NodeLifeCycler) removeUpdatingAnnotation(node *core_v1.Node) error {
+	copy := node.DeepCopy()
+	delete(copy.Annotations, AnnotationUpdateTimestamp)
+	err := lc.patch(node, copy)
+	if err != nil {
+		errors.Wrap(err, "failed to remove updating annotation")
+	}
+	return nil
+}
+
+func (lc *NodeLifeCycler) patch(old, new *core_v1.Node) error {
```

Comment:
This is fairly complicated.
I thought doing it more straightforward like this:
```
patch := []byte(`[{"op":"remove","path":"/metadata/annotations/testing"}]`)
pod, err = client.Pods(namespace).Patch(pod.Name, types.JSONPatchType,
                patch)
```

Reply:
Tell me about it... 🤣 

Now using `JSONPatchType`.